great-idea/index.html

```diff
@@ -17,7 +17,82 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+    <header>
+      <div class="header-content-container">
```

Comment:
This is just personal taste but i think it would be possible to get rid of that div and just apply those styles to header container that stores the nav and logo. This is by no means "better" way, just an opinion.

Reply:
I did this because the header is fixed; I wanted the header to be `width: 100%;` whilst the content within the header is a fixed width.